cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -45,12 +83,99 @@ class CoursewareSearchIndexer(object):
     Class to perform indexing for courseware search from different modulestores
     """
 
+    # Flags for where to index the information
+    ANALYSE = 1 << 0  # Add the information to the analysed content of the index
+    PROPERTY = 1 << 1  # Add the information as a property of the object being indexed (not analysed)
+
+    # List of properties to add to the index - each item in the list is a tuple of:
+    #   1) Property name to use
+    #   2) Where to add in the index (using flags above)
+    #   3) Whether the property is found in the 'about' area of the modulestore
+    #      (True) or is a property directly on the course object
+    ABOUT_INFORMATION_TO_INCLUDE = [
+        ("advertised_start", PROPERTY, False),
+        ("announcement", PROPERTY, True),
+        ("start", PROPERTY, False),
+        ("end", PROPERTY, False),
+        ("effort", PROPERTY, True),
+        ("display_name", ANALYSE, False),
+        ("overview", ANALYSE, True),
+        ("title", ANALYSE | PROPERTY, True),
+        ("university", ANALYSE | PROPERTY, True),
+        ("number", ANALYSE | PROPERTY, False),
+        ("short_description", ANALYSE, True),
+        ("description", ANALYSE, True),
+        ("key_dates", ANALYSE, True),
+        ("video", ANALYSE, True),
+        ("course_staff_short", ANALYSE, True),
+        ("course_staff_extended", ANALYSE, True),
+        ("requirements", ANALYSE, True),
+        ("syllabus", ANALYSE, True),
+        ("textbook", ANALYSE, True),
+        ("faq", ANALYSE, True),
+        ("more_info", ANALYSE, True),
+        ("ocw_links", ANALYSE, True),
+        ("enrollment_start", PROPERTY, False),
+        ("enrollment_end", PROPERTY, False),
+    ]
+
+    @classmethod
+    def index_about_information(cls, modulestore, course):
+        """
+        Add the given course to the course discovery index
+
+        Arguments:
+        modulestore - modulestore object to use for operations
+
+        course - course object from which to take properties, locate about information
+        """
+        searcher = SearchEngine.get_search_engine(INDEX_NAME)
+        if 'DISABLE_START_DATES' not in settings.FEATURES:
```

Comment:
Shouldn't be here IMO - don't know how it got in there, must have been a mistake!